src/styles/index.scss

```diff
@@ -173,6 +167,36 @@ font-size: 20px;
 	letter-spacing: 0.2px;
 	font-size: 17px;
 }
-.view-btn:hover{
-	background: #097205;;
+.view-btn:hover {
```

Comment:
```scss
.view-btn {
  blah: blah;
  &:hover {
    blah: blah;
  }
}
```

Reply:
thanks for this...i changed all the necessary places